### ISSUE TITLE
Add Interaction mapping

### ISSUE THREAD

**Issue by DeniseSl22** [bridgedb/Goliath#13]
Gene-variant mappings are probably too large (and not part of webservice I believe, right @fehrhart ?). But we could have interaction mappings also run from this website... so mapping from rhea to Kegg-reaction, MetaCyc etc. I definitely see this as an enhancement, so not needed for first version of website. And I'm not sure if the interaction mapping is part of the webservice calls, @egonw do you know more about this?

**Comment by fehrhart** [comment]
True, they are not part of the web service - yet...
If they are too large for this or not, I dont know for sure - @nunogit maybe you can give an answer?
The actual and archived files are here in this figshare collection: https://figshare.com/collections/BridgeDb_Gene-to-variant_mapping_database_archive/4290308

**Comment by Chris-Evelo** [comment]
Yes, we would like to integrate that in the web service, or offer a specific one. That depends on whether the size will also affect performance.

      Best, Chris
